state/account.go

```diff
@@ -178,6 +187,16 @@ func (st *Account) PendingNonce() uint64 {
 	}
 }
 
+// AdjustNonceForFreshAccount adjust the nonce for a fresh legacy account
+func (st *Account) AdjustNonceForFreshAccount(nonce uint64) uint64 {
```

Comment:
Better to be a util function, instead of a struct function

Reply:
change the name to make it more correct